File: code/344.go
/*
 * @Author: lmio
 * @Date: 2023-02-05 16:38:22
 * @LastEditTime: 2023-02-19 23:28:52
 * @FilePath: /leetcode/code/344.go
 * @Description:344.反转字符串
 */
package code

func ReverseString(s []byte){
	left := 0
	right := len(s)-1
	for left < right {
		s[left], s[right] = s[right], s[left]
		left++
		right--
	}
}

File: code/344.py
'''
Author: lmio 2091319361@qq.com
Date: 2023-08-07 12:15:25
LastEditors: lmio 2091319361@qq.com
Description: 344. 反转字符串
'''

from typing import List


class Solution:
    def reverseString(self, s: List[str]) -> None:
        left, right = 0, len(s)-1
        while left < right:
            s[left], s[right] = s[right], s[left]
            left += 1
            right -= 1